docs/route.md

```diff
@@ -116,7 +116,7 @@ val users =
 ### Endpoints
 
 **Important:** endpoints are deprecated in 0.7.0 (and will be removed in 0.8.0) in favour of
-[Coproduct Routers][router.md#coproduct-routers].
+[Coproduct Routers](router.md#coproduct-routers).
```

Comment:
As @travisbrown mentioned, it should be `route.md`. @yanana could you please update also fix it?